fragments/shared/validation/success_on_first_boot.ks

```diff
@@ -0,0 +1,59 @@
+# Report success if no errors have been reported on the first boot.
+
+%post
+
+# Create a systemd service.
+cat > /etc/systemd/system/kickstart-test.service << EOF
+
+[Unit]
+Description=The kickstart test service
+After=initial-setup.service
+After=multi-user.target
+After=graphical.target
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh /usr/libexec/kickstart-service.sh
+
+[Install]
+WantedBy=graphical.target
+WantedBy=multi-user.target
+
+EOF
+
+# Create a script with the actual test. Print errors to stdout.
+# IMPORTANT: This file should be rewritten in tests!
+touch /usr/libexec/kickstart-test.sh
+
+# Create a script for the service
+cat > /usr/libexec/kickstart-service.sh << 'EOF'
+
+# Check error messages in the syslog.
+error_messages="$(/bin/sh /usr/libexec/kickstart-test.sh)"
+
+if [[ ! -z "${error_messages}" ]]; then
+    echo "*** System has started with errors:" >> /root/RESULT
+    echo "${error_messages}" >> /root/RESULT
+fi
+
+# Validate the results.
+if [ ! -f /root/RESULT ]; then
+    echo SUCCESS > /root/RESULT
+fi
+
+# Print the results.
+cat /root/RESULT
+
+# Make the syslog accessible after the test.
+# FIXME: Find a better way to do this.
+journalctl >> /var/log/anaconda/syslog
```

Comment:
I don't mind this method but it would be great to have a separator to find out where the old ends and what is the new part.

Reply:
`journalctl` prints by default `-- Logs begin ...` on the first line.